frontend/device/android/device.lua

```diff
@@ -445,6 +453,19 @@ function Device:untar(archive, extract_to)
     return android.untar(archive, extract_to)
 end
 
+function Device:install()
+    local UIManager = require("ui/uimanager")
+    local ConfirmBox = require("ui/widget/confirmbox")
+    UIManager:show(ConfirmBox:new{
+        text = _("Update ready, do you want to install?"),
```

Comment:
```suggestion
        text = _("Update ready, do you want to install it now?"),
```

Reply:
Is there a reason not to want to use our existing (already translated etc.) text?

Reply:
The one in https://github.com/koreader/koreader/blob/master/frontend/ui/otamanager.lua#L54 is incorrect in android.

But maybe we can hide platform details an use the same prompt everywhere.

Reply:
Oh right, I forgot the exact phrasing.

Indeed, we should probably go for something more generic like:

```
Update is ready. Install it now?

Cancel              Install
```